### PR TITLE
Subpages

### PR DESCRIPTION
**Pull request by toddlamothe**
Re-added some lost content by creating subpages linked from the main readme

### CODE REVIEW

**Review by smaraf — APPROVED** [review]
This looks great, thanks for bringing back some of the context!

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Pending code reviews represent blocked threads of execution. To prevent getting
 ## Quality
 
 - prioritize quality over speed | [the only way to go fast is to go well](http://butunclebob.com/ArticleS.UncleBob.VehementMediocrity)
+- [test, automate, test, automate](docs/automate-tests.md)
```
Comment (rallytime) [inline README.md:27]: Might be a good time to mention using linting tools or other kinds of automated robot-tasks that make it easier to review PRs?

For example, the `usm-apis` repo runs a couple of easy-automated formatters that keep the code clean. These can run as part of the commit process itself if desired. They also run a part of the CI/CD checks upon PR submission and must pass before PR merge.

https://github.com/unionstreetmedia/usm-apis/blob/main/CONTRIBUTING.md#code-style

**Review by rallytime — APPROVED** [review]
👍🏻 

I added a small suggestion since you're working on these docs, but it's certainly not a blocker.

File: README.md (hunk 1 above)
Comment (toddlamothe) [reply README.md:27]: I agree. Ok if I merge this PR and we can add linting separately?

**Pull request merged**